R/write.R

```diff
@@ -133,6 +135,12 @@ write_redcap_xlsx <- function(supertbl,
     supertbl$redcap_data,
     sheet_vals,
     function(x, y) {
+
+      # Convert period/difftime to character to address possible file corruption
+      x <- x %>%
+        mutate(across(where(is.difftime), as.character),
+               across(where(is.period), as.character))
```

Comment:
This seems fine but how were we getting `period` and `difftime` classes in the supertbl anyways?

Reply:
From Marvin's email, this was related to clock time values that weren't dates/datetimes and resulting in the corrupted files I was seeing in our test database. In our REDCapTidieR Classic test database we have `text_hms`/`text_hm`, etc. and these come out with class `"hms" "difftime"`:

![image](https://github.com/CHOP-CGTInformatics/REDCapTidieR/assets/38384694/7a3e3aad-cd1e-403b-82b2-1933fc8be9d9)

Marvin mentioned `openxlsx2` doesn't yet have support for these data types. I didn't see much in `lubridate` for checking for `hms`, but think that between `difftime` and `period` checks we should be able to capture these points.